Source/WebKit/UIProcess/API/Cocoa/WKUIDelegate.h

```diff
@@ -292,7 +292,7 @@ WK_SWIFT_UI_ACTOR
  If you do not implement this method on iOS or visionOS, the web view will match the file upload behavior of Safari. If you desire
  the web view to act as if the user selected the Cancel button on iOS or visionOS, immediately call the completion handler with nil.
  */
-- (void)webView:(WKWebView *)webView runOpenPanelWithParameters:(WKOpenPanelParameters *)parameters initiatedByFrame:(WKFrameInfo *)frame completionHandler:(WK_SWIFT_UI_ACTOR void (^)(NSArray<NSURL *> * _Nullable URLs))completionHandler WK_API_AVAILABLE(macos(10.12), ios(18.4), visionos(2.4));
+- (void)webView:(WKWebView *)webView runOpenPanelWithParameters:(WKOpenPanelParameters *)parameters initiatedByFrame:(WKFrameInfo *)frame completionHandler:(WK_SWIFT_UI_ACTOR void (^)(NSArray<NSURL *> * _Nullable URLs))completionHandler WK_API_AVAILABLE(macos(10.12), ios(18.4));
```

Comment:
Before I look at others, why is this the right thing to do? I'm also surprised at how it's so new on iOS while so old on macOS.

Reply:
And it seems like the newness on iOS is legitimate. https://commits.webkit.org/284471@main made it work on iOS family platforms.

Reply:
OK, updated the commit and manually reviewed the files that had conflicts.